### PR TITLE
Update index.erb

### PR DESCRIPTION
**Pull request by sneg55**
1. I've added meta description 
2. I've added a short description of hashrate voting and how miners could participate.

### CODE REVIEW

File: views/index.erb [hunk 1]
```diff
@@ -87,6 +87,20 @@ This vote is simply an additional signal for the community to look at. It is not
...
 </p>
 
+<h2>Hashrate Voting </h2>
+<p>
+If you are pool operator, please add PPYE to your node's extra-data field to signal for #ProgPoW or PPNO to oppose it.
+</p>    
+
+<h3>How to participate in Hashrate Voting If I'm just mining ether on a pool?<h3>
+<p>
+Ask your mining pool to support ProgPoW voting.
```
Comment (lrettig) [inline views/index.erb]: Could you please reword this to be unbiased? We want to keep this site as unbiased as possible. Thanks.

**Review by lrettig — CHANGES_REQUESTED** [review]

File: views/index.erb (hunk 1 above)
Comment (lrettig) [reply views/index.erb]: Okay, I see what you meant here -- "support ProgPoW voting" is not the same thing as "vote to support ProgPoW" -- but it could be clearer. Something like, "add support for hashrate voting" maybe?

Comment (sneg55) [reply views/index.erb]: I've reworded it to avoid "support" completely.

**Review by lrettig — APPROVED** [review]
